src/Symfony/Component/ErrorHandler/ErrorRenderer/HtmlErrorRenderer.php

```diff
@@ -40,8 +41,9 @@ class HtmlErrorRenderer implements ErrorRendererInterface
     private $logger;
 
     /**
-     * @param bool|callable $debug        The debugging mode as a boolean or a callable that should return it
-     * @param bool|callable $outputBuffer The output buffer as a string or a callable that should return it
+     * @param string|FileLinkFormatter|null $fileLinkFormat
+     * @param bool|callable                 $debug          The debugging mode as a boolean or a callable that should return it
```

Comment:
```suggestion
     * @param bool|callable                 $debug          The debugging mode as a boolean or a callable that should return it
     * @param string|FileLinkFormatter|null $fileLinkFormat
```

The annotations should have the same order as the actual parameters.

Reply:
thanks, updated.